src/main/sensors/gyro.c

```diff
@@ -1165,6 +1166,12 @@ FAST_CODE void gyroUpdate(timeUs_t currentTimeUs)
         DEBUG_SET(DEBUG_DUAL_GYRO_DIFF, 2, lrintf(gyroSensor1.gyroDev.gyroADCf[Z] - gyroSensor2.gyroDev.gyroADCf[Z]));
         break;
     }
+#else
+    gyroUpdateSensor(&gyroSensor1, currentTimeUs);
+    gyro.gyroADCf[X] = gyroSensor1.gyroDev.gyroADCf[X];
+    gyro.gyroADCf[Y] = gyroSensor1.gyroDev.gyroADCf[Y];
+    gyro.gyroADCf[Z] = gyroSensor1.gyroDev.gyroADCf[Z];
+#endif
```

Comment:
I don't understand what the diff above is showing as it's really incomplete. Looking at the full file view seems correct. But anyway, the non-dual section here also needs to set the overflow and yaw spin flags like:
```
#ifdef USE_GYRO_OVERFLOW_CHECK
    overflowDetected = gyroSensor1.overflowDetected;
#endif
#ifdef USE_YAW_SPIN_RECOVERY
    yawSpinDetected = gyroSensor1.yawSpinDetected;
#endif

Reply:
Ah, okay. Was just confusing to review initially. Overall it looks good but I'm going to try and run it on my spin rig to make sure that overflow and yaw spin detect still work as expected.